rails/app/views/layouts/react.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Terrastories</title>
+    <title>Terrastories - <%= @community.name %></title>
```

Comment:
Set this to `Terrastories: <%= @community.name %></title>`

Reply:
Pushed this update!